python-main.js

```diff
@@ -361,6 +362,15 @@ function web_editor(config) {
         return EDITOR.ACE.getFontSize();
     }
 
+    // Get the editor text area and add ARIA attribute
+    function addLabel() {
+        var textArea = document.getElementsByClassName("ace_text-input")[0];
+        var textAreaLabel = document.createAttribute("aria-label");
+        textAreaLabel.value = "text editor";
+        textArea.setAttributeNode(textAreaLabel);
+    }
+    addLabel();
```

Comment:
You can do something like `$('.ace_text-input').attr('aria-label', 'text editor')` at the end of setupEditor().

Reply:
Thanks. Much nicer : )